images/ubuntu/19.04/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/ubuntu:19.04
+
+ENV NAME=ubuntu-toolbox VERSION=19.04
+LABEL com.github.debarshiray.toolbox="true" \
+      com.redhat.component="$NAME" \
```

Comment:
`com.redhat.component` shouldn't be necessary anymore. I'd suggest removing it and filing an issue or PR if something breaks.

Moreover, it refers to the Fedora bugzilla and I doubt we will be filing bugs against the Ubuntu image on the Fedora bugzilla. :)

Reply:
:) That would be a bad idea..